ee/localserver/krypto-ec-middleware.go

```diff
@@ -153,20 +176,44 @@ func (e *kryptoEcMiddleware) Wrap(next http.Handler) http.Handler {
 		var cmdReq v2CmdRequestType
 		if err := json.Unmarshal(challengeBox.RequestData(), &cmdReq); err != nil {
 			traces.SetError(span, err)
-			level.Debug(e.logger).Log("msg", "unable to unmarshal cmd request", "err", err)
+			e.slogger.Log(r.Context(), slog.LevelError,
+				"unable to unmarshal cmd request",
+				"err", err,
+			)
+
 			w.WriteHeader(http.StatusUnauthorized)
 			return
 		}
 
+		// set the kolide session id if it exists, this also the saml session id
```

Comment:
I'm not sure kolide session id and saml session id are the same.

Reply:
I ask Chris and he confirmed it was the saml id

Reply:
They might be the same now, but I'm not sure they'll be the same later. I'm suggesting we drop the comment about saml session id.